apps/go-mod-validator/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import { validateDependency } from "./github";
+import { getDependenciesMap } from "./deps";
+import * as core from "@actions/core";
+import minimist from "minimist";
+import { Octokit } from "octokit";
+
+const smartContractKitPrefix = "github.com/smartcontractkit";
+
+function getContext() {
+  const argv = minimist(process.argv.slice(2));
+  const { local, tokenEnv, goModDir } = argv;
+
+  const dir = local ? goModDir || `"$(pwd)"` : core.getInput("go-mod-dir");
+
+  const githubToken = local
+    ? process.env[tokenEnv] || ""
+    : core.getInput("github-token");
+
+  return { goModDir: dir, octokitClient: new Octokit({ auth: githubToken }) };
+}
+
+async function run() {
+  const { goModDir, octokitClient } = getContext();
+
+  // get dependencies from go.mod file
+  let dependenciesMap: Map<string, any> = new Map();
+  try {
+    dependenciesMap = getDependenciesMap(goModDir);
+  } catch (err) {
+    core.info(`failed to get dependencies, err: ${err}`);
+    core.setFailed(`failed to get dependencies`);
+  }
+
+  // <failed-dependency, error-string>
+  const validationFailedDependencies: Map<string, string> = new Map();
+
+  // Verify each of the dependencies
+  for (const [file, dependencies] of dependenciesMap.entries()) {
+    for (let dependency of dependencies) {
```

Comment:
```suggestion
    for (const dependency of dependencies) {
```

Best to use const whereever possible